### PR TITLE
feat: added grid

### PR DESCRIPTION
**Pull request by nakulsharma15**
Created 2 types of Grid Layout:

1. Two Column Grid
index-1

2. Three Column Grid
index-2

### CODE REVIEW

**Review by nishasen — CHANGES_REQUESTED** [review]
Good clean code

**Review by nishasen — APPROVED** [review]
Good clean code

File: stylesheets/components/grid.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    gap: 1rem;
+}
+
+.grid-3 {
+    grid-template-columns: repeat(3, 1fr);
+    gap: 1.5rem;
+}
+
+.grid-item {
+    border: 1px solid var(--black);
```
Comment (shashidwivedi) [inline stylesheets/components/grid.css:18]: Do not write this border here in the component. You may want to use this grid component without it. And for showcasing the grid component on your CL page, you can add this border in the CSS file meant for the page.

**Review by shashidwivedi — APPROVED** [review]
The rest looks good

File: stylesheets/components/grid.css (hunk 1 above)
Comment (nakulsharma15) [reply stylesheets/components/grid.css:18]: Ok, I'll do that!

**Pull request merged**